types/postcss-prefix-selector/postcss-prefix-selector-tests.ts

```diff
@@ -1,4 +1,4 @@
 import postcssPrefixSelector from 'postcss-prefix-selector';
```

Comment:
```suggestion
import postcssPrefixSelector ='require(postcss-prefix-selector');
```
IMO this is old school CJS module, with direct function symbol export:
https://github.com/RadValentin/postcss-prefix-selector/blob/master/index.js#L1

Reply:
that was the syntax i was looking for :) thanks for the suggestion, this did the trick